docs/docusaurus.config.ts

```diff
@@ -1,6 +1,12 @@
 import {themes as prismThemes} from 'prism-react-renderer';
 import type {Config} from '@docusaurus/types';
 import type * as Preset from '@docusaurus/preset-classic';
+import DagsterVersions from './dagsterVersions.json';
+
+const DagsterVersionsDropdownItems = Object.entries(DagsterVersions).splice(
```

Comment:
Clever!

Reply:
This is how Docusaurus does it in their own docs, so all credit to them lol